ci_config.json

```diff
@@ -72,6 +72,13 @@
       "binutils-dev"
     ]
   },
+  "aklomp-base64": {
+    "_comment": "The test program has a bug that causes it to fail with a SIGILL on x86 platforms without avx512 (all CI runners)",
```

Comment:
Has this been reported upstream?  At minimum we should link to the issue report so it's possible to follow up.  Is there any way to work around the bug from `meson.build` without patching the source?